client/src/App.js

```diff
@@ -1,8 +1,21 @@
 import React from "react";
 import { Redirect, Route, Switch, Link } from "react-router-dom";
 
+import { InteractiveExample } from "./ingredients/interactive-example";
+import { Attributes } from "./ingredients/attributes";
+import { Examples } from "./ingredients/examples";
+
 const LOCALES = ["en-US"];
 
+// Eventually these should be available directly from the documentJSON
```

Comment:
That comment will only be relevant for a matter of hours, right?

Reply:
Yes, it will be relevant until we update this again to resolve the breakage caused by merging https://github.com/mdn/stumptown-experiment/issues/19.